deps/llvm.mk

```diff
@@ -58,6 +58,12 @@ LLVM_CPPFLAGS :=
 LLVM_LDFLAGS :=
 LLVM_CMAKE :=
 
+LLVM_CFLAGS :=
+LLVM_CXXFLAGS :=
+LLVM_CPPFLAGS :=
+LLVM_LDFLAGS :=
+LLVM_CMAKE :=
+
```

Comment:
This looks to be an unrelated change

Reply:
@vtjnash is this actually important in this PR?

Reply:
Not really ... and I just realized this is already in the Makefile right above it. Apparently this was just a rebase mistake – whereas I thought it was something I had done locally.